build.rs

```diff
@@ -32,7 +32,6 @@ fn main() {
     let bindings = include_args
         .fold(bindings, |bindings, arg| bindings.clang_arg(arg))
         .header("src/wrapper.h")
-        .link("drm")
```

Comment:
Why is this no longer necessary?

Reply:
The option was removed https://github.com/rust-lang/rust-bindgen/commit/8e3e585b6efc6771cac8a9fa6ef9f9310c3d0f08

Reply:
How do we now make sure to load `libdrm.so.123` at runtime? (Sorry, my Rust is rusty, haven't been looking at this code in a while.)

Reply:
Just did some digging and looks like we could use the link attribute, but also `println!("cargo:rustc-link-lib=drm")` should also work